README.md

```diff
@@ -1,8 +1,6 @@
 # CAPI + Rancher = :cupid:
 
-A proof-of-concept project looking at various aspects of making Rancher :heart: Cluster API.
-
-> As a proof-of-concept this is doesn't in anyway indicate a future Rancher strategy and you use this at your own risk, there is no support! The code may be a bit rubbish as well.
+A project looking at various aspects of making Rancher :heart: Cluster API.
 
 ## What is covered in this PoC?
```

Comment:
Should we remove the PoC mention here as well, like we did already above?

Reply:
yes, I will update the PR

Reply:
done